nodejs/test/layer.tap.js

```diff
@@ -17,7 +17,8 @@ tap.test('Layer tests', (t) => {
     helper = utils.TestAgent.makeInstrumented() 
     const newrelic = helper.getAgentApi() 
         ;({ handler, getHandlerPath } = proxyquire('../index', {
-      newrelic
+      newrelic,
+      'test/fixtures/esm/handler': null // necessary so that we can test what happens if require will not work
```

Comment:
In testing, index.js catches the error and awaits import for ESM...whether or not the `test/fixtures/esm/handler` is defined as null for proxyquire. We could likely just restore this to the original.